hpat/str_arr_ext.py

```diff
@@ -106,7 +106,7 @@ def __init__(self, dmm, fe_type):
     ('meminfo', types.MemInfoPointer(str_arr_payload_type)),
 ]
 
-
+make_attribute_wrapper(StringArrayType, 'null_bitmap', 'null_bitmap')
```

Comment:
is any way to avoid using this?

Reply:
No, we dont have another way.
Also there is an idea in StringArrayType to create an overload method that would implement isna and which we could pull from the overload for the series.